probing/README.md

```diff
@@ -1,4 +1,81 @@
-# Edge Probing Utils
+# Edge Probing
+
+This is the main page for [What do you learn from context? Probing for sentence structure in contextualized word representations](https://openreview.net/forum?id=SJzSgnRcKX), a.k.a. "Edge Probing"
+
+## Getting Started
+
+First, follow the set-up instructions for `jiant`: [Getting Started](../README.md#getting-started). In particular, you'll need to set the following environment variables:
+- `JIANT_PROJECT_PREFIX` to wherever you want experiments to be saved (like 
+  `$HOME/exp`)
+- `JIANT_DATA_DIR` to the directory where you'll download the edge probing data 
+  (like `$HOME/jiant_data`)
```

Comment:
Suggest that they use the customizable path setup script?

Reply:
Fixed the mention on the main page.